apps/backoffice-v2/src/common/hooks/useZodSearchParams/utils/deep-null-to-string.ts

```diff
@@ -0,0 +1,15 @@
+export const deepNullToString = (value: unknown) => {
```

Comment:
Even with `qs` flags like `strictNullHandling` true/false and `skipNulls` at most I could get `null` to result in an empty string.